.github/workflows/publish-chromatic.yml

```diff
@@ -9,7 +9,7 @@ on:
       - packages/react-composites/**
       - packages/communication-react/**
   push:
-    branches: [main]
+    branches: [main, beta-candidate]
```

Comment:
my vote here would be we just use `release`
So this release branch would be: `release/1.0.0-beta.2`
Thoughts?

This, with small update to set the correct branchname and overwrite an existing branch with that name in the case we want to choose a different alpha candidate, would then be all we need to run to create new beta candidates: https://github.com/Azure/communication-ui-library/blob/main/.github/workflows/npm-release-bump.yml

Reply:
There are two workflows to consider:
* We chose a new alpha tag to create a new beta candidate off of
* We merge changes onto a beta candidate branch to fix critical bugs.

We want to automate setup for bug-bashing in both these cases.